vignettes/MDA.Rmd

```diff
@@ -40,119 +57,263 @@ simparams <- get_parameters(
     h = c(0.173743, -0.0730962, -0.116019)
   )
 )
-simparams <- set_equilibrium(simparams, starting_EIR)
-simparams <- set_drugs(simparams, list(SP_AQ_params))
-
-# Plotting functions
-plot_prevalence <- function(output) {
-  ggplot(output) + geom_line(
-    aes(x = timestep, y = (n_detect_730_3650/n_730_3650))) + 
-    labs(x = "timestep", y = "PfPR2-10")
-}
 
-plot_state_counts <- function(output) {
-  ggplot(
-    melt(
-      output[c(
-        'timestep',
-        'S_count',
-        'D_count',
-        'A_count',
-        'U_count',
-        'Tr_count'
-      )],
-      id.vars='timestep'
-    )
-  ) + geom_line(
-    aes(
-      x = timestep,
-      y = value,
-      group = variable,
-      color = variable
-    )
-  )
-}
+# Use the set_equilibrium() function to update the individual-based model parameters to those 
+# required to match the specified initial EIR (starting_EIR):
+simparams <- set_equilibrium(simparams, starting_EIR)
 
-add_mda_lines <- function(plot, events) {
-  plot + geom_vline(
-    data = events,
-    mapping = aes(xintercept=timestep),
-    color="blue"
-  ) + geom_text(
-    data = events,
-    mapping = aes(x = timestep, y = 0, label = name),
-    size = 4,
-    angle = 90,
-    vjust = -0.4,
-    hjust = 0
-  )
-}
 ```
 
-Then we can run the simulation for a variety of MDA strategies:
+### Interventions
 
-## MDA
+Having established a base set of parameters (`simparams`), the next step is to add the parameters specifying the MDA campaign. We first use the `get_drugs()` function to update `simparams` with the parameter values for the drug(s) we wish to simulate, in this case using the in-built parameter set for SP-AQ. Note that the `malariasimulation` package also contains in-built parameter sets for dihydroartemisinin-piperaquine (DHA_PQP_params) and artemether lumefantrine (AL_params), and users can also define their own drug parameters.
 
-This is a dose of SP-AQ to 80% of the population once a year.
+We then define the dates on which we want the MDA to occur (stored here in the vector `mda_events` for legibility), and use the `set_mda()` function to update the parameter list with our MDA campaign parameters. `set_mda()` takes as its first two arguments the base parameter list and the time steps on which to schedule MDA events. This function also requires the user to specify the proportion of the population we want to treat using `coverages`, as well as the age-range of the population we want to treat using `min_ages` and `max_ages`, *for each day of scheduled MDA*. Note that this enables users to specify individual events to have different drug. coverage, and human-target properties. The `drug` argument refers to the index of the drug we want to distribute in the MDA in the list of parameters (in this demonstration we have only added a single drug, SP-AQ, using `set_drugs()`, but we could have specified additional drugs).
```

Comment:
stray period instead of comma in second to last sentence after `drug`

Reply:
Good catch - corrected to a comma!